preferences/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from django.forms import ModelForm
+from preferences.models import Preferences
+
+class PreferencesForm (ModelForm):
+    class Meta:
+        model = Preferences
+        fields = ('Language', 'Mode')
```

Comment:
These fields should match the model fields
```suggestion
        fields = ('language', 'color_mode')
```